src/styles/blocks/catalog.scss

```diff
@@ -12,14 +12,21 @@
     align-items: center;
     box-sizing: border-box;
 
-    width: 200px;
-    height: 300px;
+    width: $card-width;
+    height: $card-height;
     border: 1px solid #ccc;
     border-radius: 5px;
-    margin-bottom: 48px;
+    margin-bottom: $margin-bottom-for-card;
+
+    transition: color 300ms linear, transform 300ms linear;
```

Comment:
I guess these properties can be made variables, but that's okey.

Reply:
I put properties in variables, now looks nice